alchemy-solidus.gemspec

```diff
@@ -14,8 +14,8 @@ Gem::Specification.new do |gem|
   gem.version       = Alchemy::Solidus::VERSION
 
   gem.add_dependency('alchemy_cms', ['>= 5.0.0', '< 6.1'])
-  gem.add_dependency('solidus_core', ['>= 2.10.0', '< 3.0'])
-  gem.add_dependency('solidus_backend', ['>= 2.10.0', '< 3.0'])
+  gem.add_dependency('solidus_core', ['>= 2.10.0', '< 4.0'])
+  gem.add_dependency('solidus_backend', ['>= 2.10.0', '< 4.0'])
```

Comment:
If we change that we also need to add 3.0 and 3.1 to the test matrix.
But we could also not make that change in this PR, because that takes more to support 3.0.
So, I think it’s best to remove the change from this PR

Reply:
Sounds good, done. I didn't know if this had any issue with #76.